pkg/node/types/nodename.go

```diff
@@ -42,6 +44,14 @@ func GetName() string {
 	return nodeName
 }
 
+func GetAbsoluteNodeName() string {
+	if option.Config.ClusterName != defaults.ClusterName {
```

Comment:
I think we should also check for an empty cluster name (this should also fix the failing unit test)
```suggestion
	if option.Config.ClusterName == "" || option.Config.ClusterName != defaults.ClusterName {
```